ovs/client.go

```diff
@@ -317,6 +317,14 @@ func SetSSLParam(pkey string, cert string, cacert string) OptionFunc {
 	}
 }
 
+// SetTCPParam configures the OVSDB connection using a TCP format ip:port
+// for use with all ovs commands
```

Comment:
Specify "ovs-vsctl" instead of "ovs", and add a period at the end of the comment.